public/app/core/utils/explore.ts

```diff
@@ -72,7 +74,7 @@ export async function getExploreUrl(args: GetExploreUrlArguments): Promise<strin
 
           return {
             // interpolate the query using its datasource `interpolateVariablesInQueries` method if defined, othewise return the query as-is.
-            ...(queryDs.interpolateVariablesInQueries?.([q], scopedVars ?? {})[0] || q),
+            ...(queryDs.interpolateVariablesInQueries?.([q], scopedVars ?? {}, args.adhocFilters)[0] || q),
```

Comment:
We are destructuring args on line 64, we should the the same for adhocFilters
```suggestion
            ...(queryDs.interpolateVariablesInQueries?.([q], scopedVars ?? {}, adhocFilters)[0] || q),
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/78339/commits/2039ebf8ad6491be1ef582e6dc8cc73fa092ee9e